File: api-project/MovieReservation/MovieReservation/DTO/Movie.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MovieReservation.DTO
{
    public class Movie
    {
        public Movie()
        {
            this.MovieSchedule = new List<Schedule>();
        }
        public int MovieID { get; set; }
        public string MovieName { get; set; }
        public string MovieImage { get; set; }
        public List<Schedule> MovieSchedule { get; set; }
    }
}

File: angular-proj/src/app/DTO/Movie.ts
import { Schedule } from "./Schedule";

export class Movie {
    MovieID: number;
    MovieName: string;
    MovieImage: string;
    MovieSchedule: Array<Schedule>;

    constructor(){
        this.MovieID = 0;
        this.MovieName = "";
        this.MovieImage = "";
        this.MovieSchedule = new Array<Schedule>();
    }
}